components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -141,6 +141,39 @@ def fileread_gen(fin, fsize, bufsize):
     fin.close()
 
 
+def countAnnotations(queryResult=[]):
+    """
+    Count the different the annotions from the
+    provided projection query result
+    """
+
+    counts = {
+        "TagAnnotation": 0,
+        "FileAnnotation": 0,
+        "CommentAnnotation": 0,
+        "LongAnnotation": 0,
+        "MapAnnotation": 0,
+        "OtherAnnotation": 0}
+
+    for r in queryResult:
+        ur = unwrap(r)
+        if ur[3] == '/basic/num/long/':
+            counts['LongAnnotation'] += int(ur[0])
+            counts['OtherAnnotation'] += int(ur[1])
+        elif ur[3] == '/basic/text/comment/':
+            counts['CommentAnnotation'] += int(ur[2])
+        elif ur[3] == '/basic/text/tag/':
+            counts['TagAnnotation'] += int(ur[2])
+        elif ur[3] == '/type/OriginalFile/':
+            counts['FileAnnotation'] += int(ur[2])
+        elif ur[3] == '/map/':
+            counts['MapAnnotation'] += int(ur[2])
+        else:
+            counts['OtherAnnotation'] += int(ur[2])
```

Comment:
Minor point: I don't think you really need all these ```int()``` calls since we already have Longs, unless that's not always the case?